docs/releases/1.9.txt

```diff
@@ -559,6 +559,11 @@ Miscellaneous
   :attr:`~django.test.SimpleTestCase.allow_database_queries` class attribute
   to ``True`` on your test class.
 
+* :attr:`~django.core.urlresolvers.ResolverMatch.app_name` was changed to
```

Comment:
I suggest to include at least the class name in the rendered version: `ResolverMatch.app_name <django.core.urlresolvers.ResolverMatch.app_name>` (or just omit the tilde and render the full path)